src/components/radio-button-group/radio-button-group.tsx

```diff
@@ -125,7 +125,7 @@ export class RadioButtonGroup {
   /**
    * Fires when the component has changed.
    */
-  @Event({ cancelable: false }) calciteRadioButtonGroupChange: EventEmitter<void>;
+  @Event({ cancelable: false }) calciteRadioButtonGroupChange: EventEmitter<any>;
```

Comment:
Pretty sure this is a string, right?

Reply:
Most commonly a string, but `value` allows for anything. This is useful for environments that set props to rich data to skip mapping a string to an object or something else.

Reply:
oh interesting! I assumed this corresponded to the radio input value property